jobserver/views/workspaces.py

```diff
@@ -305,9 +305,7 @@ def get(self, request, *args, **kwargs):
         has_backends = request.user.is_authenticated and request.user.backends.exists()
 
         # should we show the admin section in the UI?
-        show_admin = (
-            can_archive_workspace or repo_is_private or can_toggle_notifications
-        )
+        show_admin = can_archive_workspace or can_toggle_notifications
```

Comment:
thought: I have a very limited understanding of roles and permissions. But, if I look at the current `roles.py`, it looks as if only `ProjectDeveloper` has either of `workspace_archive` or `workspace_toggle_notifications`. No other role does.

It does seem a little messy that we're checking for either of these two permissions instead of a single role. But I guess that type of duplication could be cleaned up in the forthcoming work on improving roles and permissions.

(It's fine to leave this for now; it's more a general observation on the current state of roles and permissions.)

Reply:
I had similar thoughts and backed out of any major changes on the basis we're taking a more holistic look at this and it's likely to change more majorly quite soon